exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/TableBuilder.java

```diff
@@ -56,6 +47,30 @@ public TableBuilder(final String[] columns) {
     sb.append("</tr>\n");
   }
 
+  /**
+   * Representation of a millisecond duration in a short readable text
+   * @param millis  Duration in milliseconds
+   * @return        Human-Readable Duration Time
+   */
+  public String shortDurationFormat(long millis) {
```

Comment:
Used elsewhere? If so, consider making static (so we don't need an instance.) If only used here, consider making private.

In fact, maybe move this out to a utility class. This logic can be used anywhere we want to display a duration such as in logs, in tests, etc.